katas/kata_adalid/lang.js

```diff
@@ -0,0 +1,112 @@
+const antMoves = (large = undefined, steps = undefined, sense = '') =>{
```

Comment:
Also, please remove folder `.idea` and include `.idea/` line to `.gitignore` file so those files are not tracked. Those are files managed by your editor that will constantly change and are not needed for the project, so they should be ignored by git